### PR TITLE
Bump Go and tools deps

### PR DESCRIPTION
**Pull request by AlekSi**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,8 +1,10 @@
 # Contributing
 
 The dance tool and tests run on the host; macOS, Linux, and Windows are expected to work.
-Databases under test (FerretDB and MongoDB) may be running on the host or inside Docker; Docker Compose configuration is provided for convenience but not required.
-In particular, the FerretDB development cycle (fix-compile-run-dance) is faster with it running on the host as it does not involve Docker image building or PostgreSQL restarts.
+Databases under test (FerretDB and MongoDB) may be running on the host or inside Docker;
+Docker Compose configuration is provided for convenience but not required.
+In particular, the FerretDB development cycle (fix-compile-run-dance) is faster with it running on the host
```
Comment (b1ron) [inline CONTRIBUTING.md:6]:
```suggestion
In particular, the FerretDB development cycle (fix-compile-run-dance) is faster when running on the host
```

**Review by b1ron — APPROVED** [review]
Made a small grammatical suggestion.

File: CONTRIBUTING.md (hunk 1 above)
Comment (AlekSi) [reply CONTRIBUTING.md:6]: I think that changes the meaning: the suggested change says that the development cycle is faster when run on the host – but that's the only way we support it. The original sentence tries to say that the development cycle is faster when _FerretDB itself_ runs on a host.

Comment (b1ron) [reply CONTRIBUTING.md:6]: I understand.

**Review by b1ron — APPROVED** [review]

**Pull request merged**